src/main/java/frc/robot/Constants.java

```diff
@@ -86,6 +87,10 @@ public final Boolean isEnabled() {
   // TODO: Replace 0.69 with actual track width in meters and run characterization on real robot
 
   // Encoder constants
+
+  // Controller modes
+  public static final boolean testMode = true;
```

Comment:
testMode is pretty vague. Maybe we could set it based on the number of controllers plugged in?